test/Microsoft.AspNetCore.Mvc.FunctionalTests/TagHelpersTest.cs

```diff
@@ -87,6 +87,42 @@ public class TagHelpersTest :
 #endif
         }
 
+        [Fact]
+        public async Task ReregisteringAntiforgeryTokenInsideFormTagHelper_DoesNotAddDuplicateAntifogeryTokenFields()
```

Comment:
If you're going to misspell it, you might as well misspell it right.

![image](https://cloud.githubusercontent.com/assets/1430011/15231348/684c1bfe-184f-11e6-8e65-df5f896fe0c1.png)